src/intervals/arithmetic.jl

```diff
@@ -50,12 +50,15 @@ end
 const ≺ = strictprecedes # \prec
 
 
-# zero, one
+# zero, one, typemin, typemax
 zero(a::Interval{T}) where T<:Real = Interval(zero(T))
 zero(::Type{Interval{T}}) where T<:Real = Interval(zero(T))
 one(a::Interval{T}) where T<:Real = Interval(one(T))
 one(::Type{Interval{T}}) where T<:Real = Interval(one(T))
-
+typemin(::Type{Interval{T}}) where T<:AbstractFloat = wideinterval(typemin(T))
+typemax(::Type{Interval{T}}) where T<:AbstractFloat = wideinterval(typemax(T))
+typemin(::Type{Interval{T}}) where T<:Integer = Interval(typemin(T))
```

Comment:
The package is not supposed to be used with integer intervals, but I guess no harm is done!